pkg/k8s/util.go

```diff
@@ -2,13 +2,22 @@ package k8s
 
 import (
 	"embed"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
 	"os"
+	"path/filepath"
+	ctrl "sigs.k8s.io/controller-runtime"
 
 	"github.com/cnoe-io/idpbuilder/pkg/util"
 	"k8s.io/apimachinery/pkg/runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+var (
+	setupLog = ctrl.Log.WithName("k8s")
```

Comment:
I would rather not produce log messages in utility functions. Let's wrap the error instead. e.g. `fmt.Errorf("Error creating kubernetes client: %w"`

Reply:
ok

Reply:
Fixed with: 8158551